api/swagger-spec/v1.json

```diff
@@ -15402,7 +15402,7 @@
       "type": "string",
       "method": "GET",
       "summary": "connect GET requests to proxy of Service",
-      "nickname": "connectGetNamespacedServiceProxy",
+      "nickname": "connectGetNamespacedServiceProxyWithPath",
```

Comment:
It's unfortunate that we have to split out every request to the proxy separately by verb. :/